app/policies/repo_policy.rb

```diff
@@ -0,0 +1,31 @@
+module Pushbit
+  class RepoPolicy < Policy
+    def create?
+      false
+    end
+
+    def read?
+      record.public? || user.repos.include?(record)
```

Comment:
This is probably fine for now, just a note that as this is the cached repo if the repo is made private then this will be out of sync and potentially allow for the viewing of private repos